clients/extension/src/pages/transaction/index.tsx

```
@@ -657,7 +674,7 @@ const Component = () => {
               <div className="card">
                 <div className="header">
                   <span className="heading">{`${t('sign_transaction')} (${step}/${fastSign ? 5 : 4})`}</span>
-                  <span className="action">
+                  <span className="action" onClick={handleClose}>
```

ESLint

[error] Visible, non-interactive elements with click handlers must have at least one keyboard listener.

(jsx-a11y/click-events-have-key-events)